js/src/setup-mc/setup-stepper/setup-free-listings/shipping-time/shipping-time-setup/countries-time-input-form/countriesTimeInput.test.js

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
Thanks for adding tests. 

❓ 

Is there a reason to have `countriesPriceInput.test.js` and `countriesTimeInput.test.js` in separated files?

All the mocks and external deps are duplicated between both files. I'd consider to use only one file `tests/countriesInput.test.js` and test there both.

Besides the dependencies  the tests themselves are kind of duplicated and both are using same logic

Reply:
The idea was to keep different files to make easier the addition of other specific tests for each component. If you like I could join both tests in one file and if we need to add more tests to these components in the future, we could:
 
- Continuing adding the tests in the same file
- Or create a new file depending on the logic and component that we are going to test.

Reply:
Didn't check all the code yet but I can see how `useGetRemainingCountryCodes` and `useGetRemainingCountryCodes`  are 99% duplicated code (only one line changes between them), same goes with all the files in the modals (add & edit) and maybe more...  (TBD)

This is totally out scope of this issue but feels for me that a refactor is needed here. 

Since I don't wanna block the PR and this is more likely a generalised problem in the component I will open a separated issue regarding the refactor (Including this tests). 

Thoughts?

Reply:
I agree that shippings-rates & shipping-times are components that seem to have the same logic, consequently, duplicating code and logic. Let's open the issue where we can discuss if it is needed to refactor those two components. Thanks!